src/curve/LinearIncreasingCurve.sol

```diff
@@ -143,33 +140,40 @@ contract LinearIncreasingEscrow is
                               CURVE BIAS
     //////////////////////////////////////////////////////////////*/
 
+    /// @notice Rounds `_elapsed` to maxTime if it's greater, otherwise returns `_elapsed`.
+    function boundElapsedMaxTime(uint256 _elapsed) private view returns (uint256) {
+        uint256 MAX_TIME = maxTime();
+        return _elapsed > MAX_TIME ? MAX_TIME : _elapsed;
+    }
+
     /// @notice Returns the bias for the given time elapsed and amount, up to the maximum time
     function getBias(uint256 timeElapsed, uint256 amount) public view returns (uint256) {
         int256[3] memory coefficients = _getCoefficients(amount);
-        return _getBias(timeElapsed, coefficients[0], coefficients[1]);
+        return _getBias(boundElapsedMaxTime(timeElapsed), coefficients[0], coefficients[1]);
```

Comment:
Must make a note to regression test that this returns the same value pre and post upgrade, for all values of a lock between now and maturity